### DISCUSSION TITLE
Add Greenplum database as supported datasource

### DISCUSSION THREAD

**Discussion by mpalfi** [great-expectations/great_expectations#4178] [Request For Comment]
**Is your feature request related to a problem? Please describe.**
Full support of Greenplum DB as datasource.

**Comment by talagluck** [comment]
Hi @mpalfi! Thank you for raising this issue.

Would you please let us know some of the areas where the current implementation does not work properly for Greenplum?

Realistically, we will not be able to prioritize internal development for a Greenplum datasource, but we can absolutely support a community contribution. Please let us know if you'd be interested in that!
